### PR TITLE
Render links (@see, link and linkplain) that are not URLs as links instead of text.

### PR DESCRIPTION
**Pull request by mudsam**
Fixes issue #564 
Added new test case and updated existing to match links instead of text.

### CODE REVIEW

**Comment by mudsam** [comment]
I'm failing a couple of PC test cases, taking a look.

File: mtags/src/main/scala/scala/meta/internal/docstrings/MarkdownGenerator.scala [hunk 1]
```diff
@@ -104,9 +107,14 @@ object MarkdownGenerator {
...
       if (c.see.nonEmpty)
         "\n**See**\n" + c.see
-          .map(
-            body => "- [" ++ blocksToMarkdown(body.blocks).trim + "]()"
-          )
+          .map { body =>
+            // If we already have a link, just render it
+            // otherwise create a link from the given text
+            blocksToMarkdown(body.blocks).trim match {
+              case LinkPattern(text) => s"- $text"
```
Comment (olafurpg) [inline mtags/src/main/scala/scala/meta/internal/docstrings/MarkdownGenerator.scala]: Is it possible to move this regexp to the parser? I expected the link to be encoded in the `Comment` AST, it seems a bit unusual to do mini-parsing inside the printer

Comment (mudsam) [reply mtags/src/main/scala/scala/meta/internal/docstrings/MarkdownGenerator.scala]: Yes, makes sense. I'll try to move it into the parser.

File: mtags/src/main/scala/scala/meta/internal/docstrings/MarkdownGenerator.scala [hunk 2]
```diff
@@ -104,9 +107,14 @@ object MarkdownGenerator {
...
+              case text => s"- [$text]($text)"
```
Comment (laughedelic) [inline mtags/src/main/scala/scala/meta/internal/docstrings/MarkdownGenerator.scala]:
```suggestion
              case text => s"- [`$text`]($text)"
```

How about this pattern? It should look like a verbatim text which is also a link.

**Comment by olafurpg** [comment]
@mudsam Please reopen this PR once you have time to look into this! Closing for now to clear the PR queue

**Pull request closed without merging**